tools/istio-iptables/pkg/cmd/run_test.go

```diff
@@ -0,0 +1,360 @@
+	ipv4Range := NetworkRange{
+		IsWildcard: true,
+		IPNets:     nil,
+	}
+	iptConfigurator.handleInboundIpv4Rules(ipv4Range)
+	actual := utils.FormatIptablesCommand(iptConfigurator.iptables.BuildV4())
+	expected := []string{
+		"iptables -t nat -N ISTIO_OUTPUT",
+		"iptables -t nat -A ISTIO_OUTPUT -j ISTIO_REDIRECT",
+	}
+	if !reflect.DeepEqual(actual, expected) {
+		t.Errorf("Output mismatch.\nExpected: %#v\nActual: %#v", expected, actual)
+	}
+}
+
+func TestHandleInboundIpv4RulesWithWildcardWithKubeVirtInterfaces(t *testing.T) {
+	cfg := constructConfig()
+	iptConfigurator := NewIptablesConfigurator(cfg)
+	ipv4Range := NetworkRange{
+		IsWildcard: true,
+		IPNets:     nil,
+	}
+	iptConfigurator.cfg.KubevirtInterfaces = "eth1,eth2"
+	iptConfigurator.handleInboundIpv4Rules(ipv4Range)
+	actual := utils.FormatIptablesCommand(iptConfigurator.iptables.BuildV4())
+	expected := []string{
+		"iptables -t nat -N ISTIO_OUTPUT",
+		"iptables -t nat -A ISTIO_OUTPUT -j ISTIO_REDIRECT",
+		"iptables -t nat -I PREROUTING 1 -i eth1 -j ISTIO_REDIRECT",
+		"iptables -t nat -I PREROUTING 1 -i eth2 -j ISTIO_REDIRECT",
+	}
+	if !reflect.DeepEqual(actual, expected) {
+		t.Errorf("Output mismatch.\nExpected: %#v\nActual: %#v", expected, actual)
+	}
+}
+
+func TestHandleInboundIpv4RulesWithIpNets(t *testing.T) {
+	cfg := constructConfig()
+	iptConfigurator := NewIptablesConfigurator(cfg)
+	_, ipnet, _ := net.ParseCIDR("10.0.0.0/8")
+	ipv4Range := NetworkRange{
+		IsWildcard: false,
+		IPNets:     []*net.IPNet{ipnet},
+	}
+	iptConfigurator.handleInboundIpv4Rules(ipv4Range)
+	actual := utils.FormatIptablesCommand(iptConfigurator.iptables.BuildV4())
+	expected := []string{
+		"iptables -t nat -N ISTIO_OUTPUT",
+		"iptables -t nat -A ISTIO_OUTPUT -d 10.0.0.0/8 -j ISTIO_REDIRECT",
+		"iptables -t nat -A ISTIO_OUTPUT -j RETURN",
+	}
+	if !reflect.DeepEqual(actual, expected) {
+		t.Errorf("Output mismatch.\nExpected: %#v\nActual: %#v", expected, actual)
+	}
+}
+
+func TestHandleInboundIpv4RulesWithIpNetsWithKubeVirtInterfaces(t *testing.T) {
+	cfg := constructConfig()
+	iptConfigurator := NewIptablesConfigurator(cfg)
+	_, ipnet, _ := net.ParseCIDR("10.0.0.0/8")
+	ipv4Range := NetworkRange{
+		IsWildcard: false,
+		IPNets:     []*net.IPNet{ipnet},
+	}
+	iptConfigurator.cfg.KubevirtInterfaces = "eth1,eth2"
+	iptConfigurator.handleInboundIpv4Rules(ipv4Range)
+	actual := utils.FormatIptablesCommand(iptConfigurator.iptables.BuildV4())
+	expected := []string{
+		"iptables -t nat -N ISTIO_OUTPUT",
+		"iptables -t nat -I PREROUTING 1 -i eth1 -d 10.0.0.0/8 -j ISTIO_REDIRECT",
+		"iptables -t nat -I PREROUTING 1 -i eth2 -d 10.0.0.0/8 -j ISTIO_REDIRECT",
+		"iptables -t nat -A ISTIO_OUTPUT -d 10.0.0.0/8 -j ISTIO_REDIRECT",
+		"iptables -t nat -A ISTIO_OUTPUT -j RETURN",
+	}
+	if !reflect.DeepEqual(actual, expected) {
+		t.Errorf("Output mismatch.\nExpected: %#v\nActual: %#v", expected, actual)
+	}
+}
+
+func TestSeparateV4V6WithWildcardCidr(t *testing.T) {
```

Comment:
Capitalize the `CIDR` acronym, and this should be named `CIDR prefix` to be correct:
```suggestion
func TestSeparateV4V6WithWildcardCIDRPrefix(t *testing.T) {
```
Same in other test cases below.

Reply:
Done